runmariadb.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/bash
+PATH=$PATH:/usr/sbin/
+DATADIR="/var/lib/mysql"
+SQL_CONF_FILE=${SQL_CONF_FILE:-"/tmp/configure-mysql.sql"}
+MARIADB_PASSWORD=${MARIADB_PASSWORD:-"change_me"}
+
+if [ ! -d "$DATADIR/mysql" ]; then
+    crudini --set /etc/my.conf mysqld max_connects 64
+    crudini --set /etc/my.conf mysqld max_heap_table_size 1M
+    crudini --set /etc/my.conf mysqld innodb_buffer_pool_size 5M
+    crudini --set /etc/my.conf mysqld innodb_log_buffer_size 512K
+
+    mysql_install_db --datadir="$DATADIR"
+
+    chown -R mysql /var/log/mariadb
+    chown -R mysql "$DATADIR"
+
+    cat > "${SQL_CONF_FILE}" <<-EOSQL
+DELETE FROM mysql.user ;
+CREATE USER 'ironic'@'localhost' identified by '${MARIADB_PASSWORD}' ;
+GRANT ALL on *.* TO 'ironic'@'localhost' WITH GRANT OPTION ;
+DROP DATABASE IF EXISTS test ;
+CREATE DATABASE IF NOT EXISTS  ironic ;
+FLUSH PRIVILEGES ;
+EOSQL
+
+    mysqld_safe --datadir=$DATADIR --user mysql --init-file /tmp/configure-mysql.sql
+else
+    mysqld_safe --datadir=$DATADIR --user mysql
+fi
+
+sleep infinity
```

Comment:
rather than sleep infinity. I think it would be better to let this main container process become mysqld_safe by changing lines 27 and 29 to start with `exec`

Reply:
In runironic.sh I didn't propose it because it seems that spawns several processes.

Reply:
I was thinking about it, I agree would be better, I'll do a quick test with that